packages/table/src/locator.ts

```diff
@@ -70,11 +70,7 @@ export class Locator implements ILocator {
     }
 
     public getWidestVisibleCellInColumn(columnIndex: number): number {
-        const cellClasses = [
-            `.${Classes.columnCellIndexClass(columnIndex)}`,
-            `.${Classes.TABLE_COLUMN_NAME}`,
-        ];
-        const cells = this.tableElement.querySelectorAll(cellClasses.join(", "));
+        const cells = this.tableElement.querySelectorAll(`.${Classes.columnCellIndexClass(columnIndex)}`);
```

Comment:
Ah, sorry, should have said. 

Because the thing returned by getElementsByClassName is not iterable using forEach, essentially. It's not an array -- I'd have to use a normal for loop to iterate over it, which seems silly.